languages/cpp/main.cpp

```diff
@@ -82,9 +82,9 @@ bool Render(std::string inputPath, Vault::Context &context)
 int main(int argc, char **ppArgv)
 {
   std::string serverPath = "https://earth.vault.euclideon.com";
-  std::string username = "Username";
-  std::string password = "Password";
-  std::string modelName = "DirCube.uds";
+  std::string username = "bwockner";
+  std::string password = "bradensVaultPassword";
```

Comment:
You need to change your password - also, please use the debugging arguments next time instead of hard coding your username, password and model. The lines directly underneath this handle it, `-u bwockner -p xxxx -m Y:\\Brisbane_30cm.uds`.

Please revert this entire file.